dashboard/assets/packages/report/src/components/Chart/Chart.utils.ts

```diff
@@ -70,7 +70,7 @@ export const createOptions = ({ plot, width }: CreateOptionsProps): Options => {
     width: width,
     height: 250,
     cursor: { sync: { key: sync.key } },
-    legend: { live: false },
+    legend: { live: true },
```

Comment:
There's been a change in the newest version related to how the legend is handled. In order to have it visible on hover, we need this option set to `true`.

https://github.com/leeoniya/uPlot/compare/1.6.30...1.6.31#diff-6f581b44485db1123ab1cc1b28b05d08b22ddb46d7ad1d19252acb9f15debf3dR2627-R2632